test/DynamoPackagesUITests/DynamoPackagesUITests.cs

```diff
@@ -1,189 +0,0 @@
-﻿using Dynamo.DynamoPackagesUI;
-using Dynamo.Extensions;
-using NUnit.Framework;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using SystemTestServices;
-using System.Windows;
-using DynamoPackagesUI.Views;
-using Dynamo.DynamoPackagesUI.Utilities;
-using Dynamo.PackageManager;
-using System.Reflection;
-using Moq;
-using Dynamo.DynamoPackagesUI.ViewModels;
-using Newtonsoft.Json;
-using Dynamo.Wpf.Extensions;
-
-namespace DynamoPackagesUITests
-{
-    [TestFixture]
-    public class DynamoPackagesUITests : SystemTestBase
```

Comment:
I saw these tests were removed in the original PR, stating they referred to the "old PackageManager". Can you extend on that?

Reply:
I am working on the test and will add it in a different PR. There are some issues related to GregClient while installing package from tests. 
The deleted tests are not being built at all and they were using a different PM client which we moved away from.